acceptance-tests/src/test/java/uk/gov/di/ipv/cri/passport/acceptance_tests/step_definitions/PassportStepDefs.java

```diff
@@ -415,4 +415,9 @@ public void errorInJsonResponse(String documentNumber) throws IOException {
     public void ExpiryNotPresentInJsonResponse(String exp) throws JsonProcessingException {
         expiryAbsentFromVC(exp);
     }
+
+    @Then("User enters expiry date as current date minus (.*) minus (.*)$")
```

Comment:
I think this makes sense because we are using named arguments in the feature but if we weren't this would be 
`User enters expiry date as current date minus 18 minus 1`
Could we makes this
`User enters expiry date as current date minus (.*) months and minus (.*) days`

Reply:
i've updated it